### ISSUE TITLE
Show connected devices.

### ISSUE THREAD

**Issue by MinekPo1** [pyamsoft/tetherfi#118]
Not sure if this is possible, but I feel like showing a list of connected devices + possibly some stats would be useful.

**Comment by pyamsoft** [comment]
Good idea, I know I get the client info in the WiFi receiver but don't do anything with it yet.

I will look to creating a Connections screen and adding basic info into the Service Notification

**Comment by pyamsoft** [comment]
Version 22 is in store testing and will bring very basic support for this.

tldr, its not great, because Android doesn't let me see too much detail about "which" device is which.
I have added a "Connections" screen which will show each connecting hotspot client's IP address when they join the network, and gives you light management tools to block connections from a client IP address.

Unfortunately, because Android does not give me any more network information as of OS 10, I am unable to associate the IP address to specific devices. For now, this is the best I can do, but will actively look for better methods in the future.

<div>
<img src="https://user-images.githubusercontent.com/7267458/227740416-9662397a-3be3-43bf-aca2-a0dbbf9720c9.png" width="200"/>

<img src="https://user-images.githubusercontent.com/7267458/227740420-53df329a-8ff0-4b2f-988b-1687c5aebfee.png" width="200" />
</div>
